patterns/1-initial/good-first-project.md

```diff
@@ -0,0 +1,84 @@
+
+## Problem
+
+While any project has the potential to benefit from being developed in InnerSource style, some are more suitable candidates than others. While starting with InnerSource, especially when trying to convince the skeptics about the utility of InnerSource (see the [Start as Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md) pattern) it is important to be able to select the more appropriate projects to have a best chance of success.
+
+Moreover, while previous works focused on the criteria for readiness for InnerSource adoption at an organizational level, there seems to be no concise and actionable guideline for InnerSource-readiness for individual projects.
+
+## Story (optional)
+
+## Context
+
+- An organization is starting or is in the early phase of InnerSource adoption, with a lot of skepticism about whether or not it'll work.
+- Selecting an inappropriate pilot project might lead to failure of the entire initiative (see the [Start as Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md) pattern). 
+- Otherwise, a team might be planning to start an InnerSource project but they do not know how prepared they are and if the project they are putting forward is appropriate or not.
+
+## Forces
+
+- Lack of knowledge about which projects stand to benefit the most from InnerSource development.
+- The team might think they are prepared for InnerSource, but might not have an idea about what challenges lie ahead and how prepared they actually are.
+- Unsure if the processes around the project are conducive to InnerSource style development.
+
+## Sketch (optional)
+
+## Solutions
+
+Based on the two previous works mentioned earlier (see [Rationale](#rationale-optional)), we hypothesize that the _fitness_ of a project can be measured by assessing whether the project has the necessary qualities to attract enough developers to build a community around it, that the tools and processes allow for collaborative development, and that the project maintainers are ready to face the challenges of InnerSource style development. We label this as the  _Project-Process-People model_. We suggest two sub-factors within each which overall provides __six__ dimensions to measure the fitness of candidate projects.
```

Comment:
If you accept my suggestion above, you can likely simply this sentence a bit as the "prior art" has already been introduced.